Tests/SwerverTests/TestData.swift

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
This needs to be removed. This is a good reminder to make sure to run your tests before committing.

Reply:
will fix that

Reply:
fixed